src/brokers/metrics/components/CardBrokerMemoryUsageMetrics/CardBrokerMemoryUsageMetrics.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react';
+import { Card, CardBody } from '@patternfly/react-core';
+import {
+  ChartMemoryUsage,
+  ChartMemoryUsageProps,
+} from '../ChartMemoryUsage/ChartMemoryUsage';
+import { useTranslation } from '../../../../i18n';
+import { CardBodyLoading } from '../CardBodyLoading/CardBodyLoading';
+import { EmptyStateNoMetricsData } from '../EmptyStateNoMetricsData/EmptyStateNoMetricsData';
+import { ChartTitle } from '../ChartTitle';
+
+export type CardBrokerMemoryUsageMetricsProps = ChartMemoryUsageProps & {
+  isInitialLoading: boolean;
+  backendUnavailable: boolean;
+};
+
+export const CardBrokerMemoryUsageMetrics: FC<
+  CardBrokerMemoryUsageMetricsProps
+> = ({
+  isInitialLoading,
+  backendUnavailable,
+  allMetricsSeries,
+  span,
+  samples,
+  fixedXDomain,
+  isLoading,
+  formatSeriesTitle,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card data-test-id={'metrics-broker-memory-usage'}>
+      {(() => {
+        switch (true) {
+          case isInitialLoading:
+            return <CardBodyLoading />;
+          case backendUnavailable:
```

Comment:
It's good that we're covering some of the non-happy paths with the initial pass.